duos_ui/lib/main.dart

```diff
@@ -17,7 +18,7 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.purple,
       ),
-      home: const LoginPage(),
+      home: const JoinPage(),
```

Comment:
Why are we changing this? I take it this was something that you change for testing. The log-on screen should be our first screen.

Reply:
changed to whatever is currently in main to merge, working on another branch for state